python/kwiver/vital/algo/algorithm_factory.cxx

```diff
@@ -52,6 +52,7 @@ static void add_algorithm( const std::string& impl_name, std::string const& desc
 
 void mark_algorithm_as_loaded( const std::string& module_name );
 static std::vector< std::string > implementation_names(const std::string& algorithm_name);
+void create_object();
```

Comment:
Where is this function being used?

Reply:
In the bindings below: https://github.com/Kitware/kwiver/blob/57cec13138c07c9b2de04edb6bb8f72975e054e5/python/kwiver/vital/algo/algorithm_factory.cxx#L96

And in some tests: https://github.com/Kitware/kwiver/blob/57cec13138c07c9b2de04edb6bb8f72975e054e5/python/kwiver/vital/tests/test_algorithm_factory.py#L84

https://github.com/Kitware/kwiver/blob/57cec13138c07c9b2de04edb6bb8f72975e054e5/python/kwiver/vital/tests/test_algorithms_common.py#L146

Reply:
Wouldn't the namespace be kwiver::vital::python then? I was confused by the use kwiver::vital:: namespace.

Reply:
Oh no you are correct. This is actually not used, as I thought I would need to write an entire new create_object function, when instead the one already defined in kwiver::vital can be used. Pushed a fix in abd3f1fdda1530772d6e50db05e4e2b594c950c1